### ISSUE TITLE
Altair Chart Rendered in Wrong Notebook in JupyterLab

### ISSUE THREAD

**Issue by sterlinm** [altair-viz/altair#1978]
I've noticed that when I use altair in JupyterLab if I open one notebook and create an altair chart, then go to a second notebook and create a different altair chart it replaces the chart from the first notebook instead of rendering it in the second notebook. I can reproduce this using the [Simple Bar Chart](https://altair-viz.github.io/gallery/simple_bar_chart.html) example from the docs.

I'm running JupyterLab 1.2.3 from my base environment and the notebooks are using a separate conda environment with altair 4.0.0.

I created a gif as that seemed the most straightforward way to show it, but I'm just running this code in two notebooks one after the other:

```python
import altair as alt
import pandas as pd

source = pd.DataFrame({
    'a': ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I'],
    'b': [28, 55, 43, 91, 81, 53, 19, 87, 52]
})

alt.Chart(source).mark_bar().encode(
    x='a',
    y='b'
)
```
![Altair Notebooks Bug Example](https://user-images.githubusercontent.com/1461273/75043862-b19a7380-548e-11ea-82c9-c3ca47d0aacf.gif)

**Comment by jakevdp** [comment]
This issue has been fixed in Altair v4.0.1 (and was in fact the reason we released that version).

**Comment by sterlinm** [comment]
Okay thanks! I checked the release notes for 4.0.0 but missed 4.0.1 somehow.
